src/pages/kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -14,43 +15,112 @@ title: Setting Up An Alert
+
+You can see all of your existing alerts by clicking `Alerts` on the main navigation menu. This will show you all the alerts in your organization by name, user, current state, and created date.
 
 ![](/assets/images/docs/gitbook/alerts.png)
 
-To set up a new alert you'll need to follow these steps:
+# Usage
+
+Create a new Alert by clicking the blue `Create` button in the navigation menu and selecting `Alert`.
+
+![](/assets/images/docs/gitbook/create-alert.png)
+
+Find the query that you would like to monitor using the search bar. Keep in mind that Alerts do not work with queries that use parameters.
+
+![](/assets/images/docs/gitbook/new-alert-query-search.png)
+
+{% callout info %}
+
+If you set an Alert for a query that returns more than one row of data, the Alert will only read the first row of data.
+
+{% endcallout %}
+
+Use the settings panel to configure your alert:
+
+![](/assets/images/docs/gitbook/alerts_settings.png)
+
+* **Value Column** is the column in your query return that you would like to watch.
+* **Op** is short for "Operation". You can choose greater than, less than, or equal to.
+* **Reference** is the absolute value that your _Value Column_ will be compared to. The most recent query return is shown as the **Value**
+* **Rearm seconds** effects how frequently you will receive notifications when your query meets the Alert criteria and does not change. See further discussion below.
+
+Finally add an [alert destination]({% link _kb/user-guide/alerts/creating-new-alert-destination.md %}) and save your alert. If you skip this step you will not receive any notifications when the alert is triggered.
+
+![](/assets/images/docs/gitbook/alert_destination.png)
+
+# Alert Status & Frequency
+
+The status of your Alert criteria is checked every time the query is executed. Alerts have three possible statuses:
+
+* `TRIGGERED` means that your query matched the criteria defined in your alert configuration. If you set your alert to trigger when the value of "cats" is greater than 1500 as long as it's above 1500 your alert is triggered.
+* `OK` means that the most recent query execution did not match the criteria defined in your alert. This doesn't mean that the Alert has not been triggered previously. If your "cats" value is now 1470 your alert will show as OK.
+* `UNKNOWN` means the query has not executed since you configured the Alert. You should see this status immediately after creating your Alert. The status will always show triggered or OK after that. If it does not change following your next query execution then review the above instructions as you have likely configured the Alert incorrectly.
```

Comment:
Alert will change to `Unknown` again if there was no data in the query result or if the query result didn't have the selected column.

Reply:
Fixed in f61077db9e4cf37d09f43873ccbce80b2cf045df